src/components/BrowserExamplesMenu/snippets/fillform.js

```diff
@@ -35,15 +35,17 @@ export default async function () {
     // to resolve.
     await Promise.all([page.waitForNavigation(), page.locator('input[type="submit"]').click()]);
 
-    const h2 = page.locator('h2');
-    const headerOK = (await h2.textContent()) == 'Welcome, admin!';
-    check(headerOK, { header: headerOK });
+    await check(page.locator('h2'), {
+      header: async (lo) => {
```

Comment:
I would skip abbreviations in example code (with some exceptions).
I assume `lo` is an abbreviation for "locator" (which resolves into an element-ish).
I suggest that we change `lo` into `element`.

Reply:
Good catch! changed it 👍